### PR TITLE
upgrade RRTMGP

### PR DESCRIPTION
**Pull request by szy21**

### CODE REVIEW

**Comment by szy21** [comment]
bors try

File: examples/hybrid/radiation_utilities.jl [hunk 1]
```diff
@@ -111,16 +111,20 @@ function rrtmgp_model_cache(
                     center_cloud_ice_water_path = RRTMGPI.field2array(
                         @. ifelse(ᶜis_top_cloud, FT(0.001) * ᶜΔz, FT(0))
                     ),
-                    center_cloud_boolean_mask = RRTMGPI.field2array(
-                        @. ᶜis_bottom_cloud || ᶜis_top_cloud
+                    center_cloud_fraction = RRTMGPI.field2array(
+                        @. ifelse(
+                            ᶜis_bottom_cloud || ᶜis_top_cloud,
+                            FT(1),
+                            FT(0) * ᶜΔz,
```
Comment (dennisYatunin) [inline examples/hybrid/radiation_utilities.jl:118]: I don't think you need `* ᶜΔz` here

**Review by dennisYatunin — APPROVED** [review]

**Review by charleskawczynski — APPROVED** [review]

**Review by sriharshakandala — APPROVED** [review]

**Pull request merged**